content/en/docs/FAQ/performance.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Performance and Scalability"
+description: "Questions about Kiali Performance measurements and improvements."
+---
+
+### What performance and scalability measurements are done?
+
+Performance tests are conducted on setups with **10**, **50**, **200**, **300**, **500**, and **800** namespaces. Each namespace contains:
+
+- 1 Service
+- 2 Workloads
+- 2 Istio configurations
+
+
+### What improvements have been made to Kiali's performance in recent versions?
+
+Performance data is collected using automated performance tests on various setups, ensuring a comprehensive evaluation of improvements.
+Since the release of Kiali v1.80, significant performance enhancements have been implemented, resulting in up to a **5x improvement** in page load times. 
+The performance improvements were achieved by reducing the number of requests made from the Kiali UI to the services. Instead of multiple requests, the process was streamlined to unify these into a single request per cluster.
+The enhanced performance significantly reduces the time users spend waiting for pages to load, leading to a more efficient and smooth user experience.
+
+**Performance Improvements Matrix Per Kiali Version And Section**
```

Comment:
I like the table much more.   Do we have the ability to add a column showing %improvements?  It's not required, this qualitative approach still indicates where the improvements can be found.

Reply:
do you mean to remove the 'improvements' column? or to add a measurements and performance difference in 'improvements' column?